pkg/webui/components/key-value-map/index.js

```diff
@@ -103,26 +103,31 @@ class KeyValueMap extends React.PureComponent {
       onBlur,
       indexAsKey,
       disabled,
+      isReadOnly,
     } = this.props
 
     return (
       <div data-test-id={'key-value-map'} className={classnames(className, style.container)}>
         <div>
           {value &&
-            value.map((value, index) => (
-              <Entry
-                key={`${name}[${index}]`}
-                name={name}
-                value={value}
-                keyPlaceholder={keyPlaceholder}
-                valuePlaceholder={valuePlaceholder}
-                index={index}
-                onRemoveButtonClick={this.removeEntry}
-                onChange={this.handleEntryChange}
-                onBlur={onBlur}
-                indexAsKey={indexAsKey}
-              />
-            ))}
+            value.map((value, index) => {
+              const readOnly = isReadOnly(value)
+              return (
+                <Entry
+                  key={`${name}[${index}]`}
+                  name={name}
+                  value={value}
+                  keyPlaceholder={keyPlaceholder}
+                  valuePlaceholder={valuePlaceholder}
+                  index={index}
+                  onRemoveButtonClick={this.removeEntry}
+                  onChange={this.handleEntryChange}
+                  onBlur={onBlur}
+                  indexAsKey={indexAsKey}
+                  readOnly={readOnly}
```

Comment:
```suggestion
                  readOnly={isReadonly(value)}
```

No need to break up the arrow short hand for that variable definition.